.github/workflows/update-go-versions.yml

```diff
@@ -0,0 +1,35 @@
+---
+name: Generate Metric files for new Go version
+
+on:
+  workflow_dispatch:
+  schedule:
+    # Runs weekly during month of February and August
+    # Reference: https://go.dev/wiki/Go-Release-Cycle
+    - cron: '0 0 * 2,8 1'
```

Comment:
The document doesn't state the actual day of the release so maybe we should run it on the last day of the month.

Still, this wouldn't be very fail-safe; what happens if the release is stalled or postponed? I don't see any downsides to running the task every month. It's a no-op if there isn't any version released.

Reply:
Agreed 👍
Let's schedule it to run on a monthly basis.